### ISSUE TITLE
"Error: Could not update the JSON file: Could not update file: Changes must be made through a pull request."

### ISSUE THREAD

**Issue by laurent22** [cla-assistant/cla-assistant#970]
## URL to the linked Repository or Organization

https://github.com/laurent22/joplin/pull/7755

## Steps to reproduce the problem

There's not specific steps, but for some reason one of the signatures is not being processed:

    https://github.com/laurent22/joplin/actions/runs/4155636396
    Run contributor-assistant/github-action@v2.2.1
      with:
        path-to-signatures: readme/cla_signatures.json
        path-to-document: https://github.com/laurent22/joplin/blob/dev/readme/cla.md
        branch: dev
        allowlist: joplinbot,renovate[bot]
        use-dco-flag: false
        lock-pullrequest-aftermerge: true
      env:
        GITHUB_TOKEN: ***
        PERSONAL_ACCESS_TOKEN: ***
    CLA Assistant GitHub Action bot has started the process
    Error: Could not update the JSON file: Could not update file: Changes must be made through a pull request.

I don't quite understand what that means - the pull request has been signed [here](https://github.com/laurent22/joplin/pull/7755#issuecomment-1426982152) like all other pull requests, but for some reason this one in particular does not work. I've also retried the job without success.

Any idea what might be the issue?

**Comment by laurent22** [comment]
In fact it seems it's now happening with all pull requests. This one too, which was signed but the action failed with same error, and signature file was not updated:

https://github.com/laurent22/joplin/pull/7757

**Comment by KharitonOff** [comment]
@laurent22 could you please create an issue in the following repo https://github.com/contributor-assistant/github-action

**Comment by laurent22** [comment]
Yes sure, I have created it here now: https://github.com/contributor-assistant/github-action/issues/131

**Comment by laurent22** [comment]
@KharitonOff, what is the official repo for this bot? It seems both of them are rather inactive at the moment?

**Comment by KharitonOff** [comment]
The repo I have mentioned above is the official one. But indeed, the activity is relatively low since this is a private project, currently driven by one person
